Sources/xcodeproj/Objects/Project/PBXProj.swift

```diff
@@ -70,10 +70,22 @@ public final class PBXProj: Decodable {
         classes = try container.decodeIfPresent([String: Any].self, forKey: .classes) ?? [:]
         let objectsDictionary: [String: Any] = try container.decodeIfPresent([String: Any].self, forKey: .objects) ?? [:]
         let objectsDictionaries: [String: [String: Any]] = (objectsDictionary as? [String: [String: Any]]) ?? [:]
-        try objectsDictionaries.forEach { reference, dictionary in
-            let object = try PBXObject.parse(reference: reference, dictionary: dictionary, userInfo: decoder.userInfo)
+        
+        let dictionary = NSDictionary(dictionary: objectsDictionaries)
+        dictionary.enumerateKeysAndObjects(options: .concurrent) { (key, obj, _) in
+            guard let reference = key as? String,
+                let dictionary = obj as? Dictionary<String, Any>
+                else { return }
```

Comment:
If the key is not a string, nor the value is a dictionary, we are skipping one entry here, which will result in a in-memory representation of the project that doesn't match the content in disk. What do you think about assuming that the types are the expected ones?

```swift
let reference = key as! String
let dictionary = obj as! Dictionary<String, Any>
```

Alternatively, we can throw an error that gives us more details about why it failed. The idea is being able to catch those as issues, instead of swallowing them.